apps/service_providers/llm_service.py

```diff
@@ -54,9 +54,9 @@ class AzureLlmService(LlmService):
 
     def get_chat_model(self, llm_model: str, temperature: float) -> BaseChatModel:
         return AzureChatOpenAI(
-            model=llm_model,
-            temperature=temperature,
-            openai_api_key=self.openai_api_key,
             openai_api_base=self.openai_api_base,
             openai_api_version=self.openai_api_version,
+            openai_api_key=self.openai_api_key,
+            deployment_name=llm_model,
+            temperature=temperature,
```

Comment:
@bderenzi this still doesn't seem to work - I think we need both the `deployment_name` and the `model`